.travis.yml

```diff
@@ -0,0 +1,16 @@
+dist: xenial
+language: php
+php:
+  - '7.1'
+  - '7.2'
+  - '7.3'
+  
+install:
+  - php composer.phar update --no-suggest
+  - php composer.phar install --no-suggest
+  
+before_script:
+  - if find . -name "*.php" ! -path "./vendor/*" -print0 | xargs -0 -n 1 -P 8 php -l | grep -v "No syntax errors detected"; then exit 1; fi
```

Comment:
Fascinating. I recently read the PHP documentation about the CLI options but I did not really take notice of the lint option... This is pretty nice, using it in combination with a shell script to check all PHP files for errors. I like it!